src/VolatilityDynamicFeeHook.sol

```diff
@@ -98,4 +101,11 @@ contract VolatilityDynamicFeeHook is BaseTestHooks {
             absTick := xor(mask, add(mask, tick))
         }
     }
+
+    function changeHookSettings(uint24 _maxFee, uint24 _fixedFee, uint256 _updateFeePeriod, uint256 _maxTickDelta) external onlyPoolManager {
```

Comment:
How will poolManager be able to call this function ? As far as I know manager can't call random functions.